src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/DataCenterFormula.kt

```diff
@@ -251,9 +252,13 @@ class DataCenterFormula private constructor(
 
         val databaseDataLocation = setupDatabase.get()
 
+        val updateJiraConfiguration =
+            if (loadBalancer is ApacheProxyLoadBalancer) listOf(loadBalancer::updateJiraConfiguration) else emptyList()
```

Comment:
;)

Reply:
@dagguh We can think about an abstraction if more cases appear in the if statement.

Reply:
Yeah, hopefully [JPERF-273](https://ecosystem.atlassian.net/browse/JPERF-273) will unblock all of that